xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -3318,6 +3318,24 @@ bool CDVDPlayer::OpenVideoStream(CDVDStreamInfo& hint, bool reset)
     }
     hint.software = true;
   }
+  else if (m_pInputStream && m_pInputStream->IsStreamType(DVDSTREAM_TYPE_PVRMANAGER))
+  {
+    // set framerate if not set by demuxer
+    if (hint.fpsrate == 0 || hint.fpsscale == 0)
+    {
+      int fpsidx = CSettings::Get().GetInt("pvrplayback.fps");
+      if (fpsidx == 0)
+      {
+        hint.fpsscale = 1000;
+        hint.fpsrate = 50000;
+      }
+      else if (fpsidx == 1)
+      {
+        hint.fpsscale = 1000;
+        hint.fpsrate = 59940;
+      }
```

Comment:
check out this comment: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/dvdplayer/DVDStreamInfo.h#L59
why hiding the information behind some formula only a few are familiar with?